.github/workflows/build-and-deploy.yml

```diff
@@ -165,6 +200,7 @@ jobs:
       - build
       - build-website
       - test-deploy
+      - test-packaging
```

Comment:
Why do we need to package to build the website?

Reply:
The 'test-packaging' job is necessary for the 'release' job, not for the 'build-website' job

Reply:
then specify it as a dependency for release, not for build-website

Reply:
It's already specified in the "needs" section of release and not build-website. 
Github with this view made a mess, + "- test-packaging" is in line 203 while build-website needs are in lines 127-128

https://github.com/Zimbrando/Alchemist/blob/911411336d985e239d5280273225195808f76ca3/.github/workflows/build-and-deploy.yml#L203